packages/lb-annotation/src/core/pointCloud/index.ts

```diff
@@ -838,21 +841,49 @@ export class PointCloud extends EventListener {
   public renderPointCloud(points: THREE.Points, radius?: number) {
     points.name = this.pointCloudObjectName;
 
-    const pointsMaterial = new THREE.PointsMaterial({
-      vertexColors: true,
-    });
+    const material = new THREE.ShaderMaterial({
```

Comment:
what about the points in pointcloud renderer?

Reply:
No need, these two types of materials can coexist. The key of update is to filter the points by shader.